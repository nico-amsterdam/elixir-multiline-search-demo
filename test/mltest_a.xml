<?xml version="1.0" encoding="utf-8"?>
<root>
	<item name"item a1">
		<definition />
		<Templates>
			<tpl name="tpl1">tpl text1</tpl>
			<tpl name="tpl2Â¨>tpl text2</tpl>
		</Templates>
		<properties />
	</item>
	<item name="item a2"><Templates><tpl name="tpl3" >tpl text3</tpl></Templates></item>
</root>
