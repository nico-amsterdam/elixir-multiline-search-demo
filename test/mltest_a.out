<Templates>
			<tpl name="tpl1">tpl text1</tpl>
			<tpl name="tpl2Â¨>tpl text2</tpl>
		</Templates>
<Templates><tpl name="tpl3" >tpl text3</tpl></Templates>
